### ISSUE TITLE
java.lang.NullPointerException: getVirtualFile(...) must not be null

### ISSUE THREAD

**Issue by sajjanm** [aws/aws-toolkit-jetbrains#4554]
After the plugin got updates, I can see the overall changeLogs of the files that I have worked on but if you try to check what has changed or try to see the history of the file, it fails. 

the error log from shown in the IntelliJ  on the editor says `getVirtualFile(...) must be not null`
Removing the plugin, everything else works

**Comment by eddy961206** [comment]
same problem here!

**Comment by aumaOne** [comment]
After updated the plugin, SQL Editor, File Search, etc is broken. After deactivating the AWS Plugins all work as expected. 

Found a lot of exceptions in the logs like:
```
2024-06-07 09:44:48,056 [ 912394]   INFO - STDERR - Caused by: java.lang.NullPointerException: getVirtualFile(...) must not be null
2024-06-07 09:44:48,056 [ 912394]   INFO - STDERR -     at software.aws.toolkits.jetbrains.services.codewhisperer.codescan.listeners.CodeWhispererCodeScanFileListener.editorCreated(CodeWhispererCodeScanFileListe
ner.kt:18)
2024-06-07 09:44:48,056 [ 912394]   INFO - STDERR -     at jdk.internal.reflect.GeneratedMethodAccessor204.invoke(Unknown Source)
2024-06-07 09:44:48,056 [ 912394]   INFO - STDERR -     at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2024-06-07 09:44:48,056 [ 912394]   INFO - STDERR -     at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2024-06-07 09:44:48,056 [ 912394]   INFO - STDERR -     at com.intellij.util.EventDispatcher.dispatchVoidMethod(EventDispatcher.java:119)
2024-06-07 09:44:48,056 [ 912394]   INFO - STDERR -     ... 66 more
2024-06-07 09:44:48,057 [ 912395] SEVERE - #c.i.o.p.Task - IntelliJ IDEA 2024.1.2  Build #IU-241.17011.79
2024-06-07 09:44:48,057 [ 912395] SEVERE - #c.i.o.p.Task - JDK: 17.0.11; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-06-07 09:44:48,057 [ 912395] SEVERE - #c.i.o.p.Task - OS: Linux
2024-06-07 09:44:48,058 [ 912396] SEVERE - #c.i.o.p.Task - Plugin to blame: Amazon Q version: 3.9-241
```

or 

```
2024-06-07 09:45:38,692 [ 963030] SEVERE - #c.i.o.f.i.FileEditorManagerImpl - CompositeException (2 nested):
------------------------------
[1]: getVirtualFile(...) must not be null
[2]: getVirtualFile(...) must not be null
------------------------------

CompositeException (2 nested):
------------------------------
[1]: java.lang.NullPointerException: getVirtualFile(...) must not be null
        at software.aws.toolkits.jetbrains.services.codewhisperer.codescan.listeners.CodeWhispererCodeScanFileListener.editorCreated(CodeWhispererCodeScanFileListener.kt:18)
        at jdk.internal.reflect.GeneratedMethodAccessor204.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at com.intellij.util.EventDispatcher.dispatchVoidMethod(EventDispatcher.java:119)
        at com.intellij.util.EventDispatcher.lambda$createMulticaster$1(EventDispatcher.java:84)
        at jdk.proxy2/jdk.proxy2.$Proxy151.editorCreated(Unknown Source)
        at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor(EditorFactoryImpl.java:228)
```

**Comment by medmunds** [comment]
See also [YouTrack #IJPL-156254](https://youtrack.jetbrains.com/issue/IJPL-156254/Create-new-branch-dialog-doesnt-open)

**Comment by aiasmartakis** [comment]
was going to add that diffing files is broken too

**Comment by byowens** [comment]
Committing files is also broken. Must remove the plugin to regain commit functions in the IDE.  This is a disruptive problem.

**Comment by badoucamara** [comment]
For information if it can help losing time for someone, Remote JVM Debugging fails also with the plugin enabled. See https://github.com/aws/aws-toolkit-jetbrains/issues/4561

**Comment by rli** [comment]
Resolved in v3.10
